Makefile

```diff
@@ -114,13 +115,24 @@ markdownlint-fix:
 pull-schemas-submodule:
 	git submodule update schemas
 
+.PHONY: pull-jsonschemas-submodule
+pull-jsonschemas-submodule:
+	git submodule update jsonschemas
```

Comment:
We could just run a wget comment to download the schemas we need instead of adding another submodule. I'm good with either approach though.

Reply:
I've removed this 2nd submodule. I think it's overkill.